README.md

```diff
@@ -151,31 +162,32 @@ please feel free to use this basic working version.
 
 ### _rotary_ workflow summary
 
-1. Perform simple long read QC using
+1. Perform simple short read QC (e.g., score, length, and adapter trimming) using [bbduk.sh from bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
```

Comment:
OK to remove "simple"? I think we are doing a fairly robust short read QC at this point.